### ISSUE TITLE
Segmentation results not good enough. Clarification regarding preprocessing code.

### ISSUE THREAD

**Issue by NishaniKasineshan** [JJGO/UniverSeg#27]
I ran through all the colab steps and it worked pretty well the sample datasets but not good with custom data provided as follows:
with WBC:
![wbc](https://github.com/JJGO/UniverSeg/assets/73964224/bd862002-85a1-4824-893f-48e21eff9cdb)
with OASIS:
![oasis](https://github.com/JJGO/UniverSeg/assets/73964224/dcb82024-520c-44e0-a9b0-9d92e59c1d74)
with my datasets(publicaly available):
![tufts](https://github.com/JJGO/UniverSeg/assets/73964224/5ad5f6a9-ba8d-463d-bb20-f2441f13a767)
 
![output](https://github.com/JJGO/UniverSeg/assets/73964224/adbd2913-9602-44a9-a3c7-73df41bdcf33)
 I have my preprocessing codes as follows to be applied on the support images and support labels:
```
def process_img(path: None):
    img = cv2.imread(path,cv2.IMREAD_GRAYSCALE)
    img = cv2.resize(img,(128,128)).reshape(1, 128, 128)
    img = img.astype(np.float32)
    return img

def process_seg(path: None):
    seg = cv2.imread(path,cv2.IMREAD_GRAYSCALE)
    seg = cv2.resize(seg,(128,128)).reshape(1, 128, 128)
    seg = seg.astype(np.float32)
    return seg
 processed_support_images = [T(process_img(image)) for image in _support_images]
 processed_support_labels = [T(process_seg(label))for label in _support_labels]

 support_images = torch.stack(processed_support_images).to(device)
 support_labels = torch.stack(processed_support_labels).to(device)
```
Is there any issue with my preprocessing code?

**Comment by NishaniKasineshan** [comment]
Actually, I had the issue with my preprocessing code. I have figured it out. Thank you for your amazing work.
